frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -1,19 +1,20 @@
-import Image, { ImageProps, ImageLoader } from 'next/image';
+import Image, { ImageProps, ImageLoader, ImageLoaderProps } from 'next/image';
 
 export function IfixitImage(props: ImageProps) {
-   let unoptimized = props.unoptimized;
+   const unoptimized = props.unoptimized;
```

Comment:
We probably don't need to specify `unoptimized` anymore, since it'll just get passed along to `<Image>` when we destructure `...props`.

Reply:
In the same vein, we could say `let loader = props.loader`, but what you have is also fine

Reply:
[61dbc35](https://github.com/iFixit/react-commerce/pull/537/commits/61dbc35a2c5a6d8f8818994c8c25225ae159ef12)